cmd/oceanbench/set.go

```diff
@@ -106,6 +107,12 @@ func writeDevices(w http.ResponseWriter, r *http.Request, msg string, args ...in
 		return
 	}
 	skey, _ := profileData(profile)
+	sandbox := false
+	file := "set/device.html"
+	if skey == 3 {
```

Comment:
Use a const, e.g. _sanboxSiteKey_

Reply:
Maybe this should be exported somewhere, since it is also used by oceancenter?

Reply:
Yes. I think it could live in `model/site.go` in lieu of another common cloud package.

Reply:
done